### PR TITLE
New boilerplate instructions

### PR DESCRIPTION
**Pull request by fenske**

### CODE REVIEW

File: .github/workflows/setup-boilerplate.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Setup boilerplate
```
Comment (fenske) [inline .github/workflows/setup-boilerplate.yml:1]: Step 1: Add this workflow.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -52,11 +52,11 @@ jobs:
         uses: joelwmale/webhook-action@master
         if: always()
         with:
-          url: https://app.devskills.co/api/assignment-auto-test-results
+          url: ${{ vars.AUTO_SCORE_WEBHOOK_URL }}
```
Comment (fenske) [inline .github/workflows/tests.yml:55]: Step 2: Update the urls in this file.

File: README.md [hunk 3]
```diff
@@ -8,30 +8,33 @@ Please agree with your hiring team regarding the tech stack choice.
 
 ### If you run into a problem
 
-Need help? Head over to [our community on GitHub](https://github.com/orgs/DevSkillsHQ/discussions/categories/help) to get assistance.
+Head over to [our community on GitHub](https://github.com/orgs/DevSkillsHQ/discussions/categories/help) to get assistance.
```
Comment (fenske) [inline README.md:11]: Step 3: Make the README changes like in this file.

**Review by fenske — COMMENTED** [review]
See what's needed to be done for the rest of the challenges.

**Comment by paliwodar** [comment]
> See what's needed to be done for the rest of the challenges.

@fenske , do you want to me propagate in the backend challenges?

**Comment by fenske** [comment]
> > See what's needed to be done for the rest of the challenges.
> 
> @fenske , do you want to me propagate this in the backend challenges?

Hi @paliwodar! Didn't realize you would be pinged. 

@Safi-Ullah is working on a task to propagate an improved boilerplate import procedure across all standard challenges.

The comment was for him.

So no action is needed from you in this case.

**Pull request merged**